http/auth/access/wrapper.go

```diff
@@ -0,0 +1,345 @@
+package access
+
+import (
+	"net/http"
+	"regexp"
+	"strings"
+
+	"github.com/centrifuge/go-substrate-rpc-client/v4/types"
+	"github.com/centrifuge/pod/config"
+	"github.com/centrifuge/pod/contextutil"
+	authToken "github.com/centrifuge/pod/http/auth/token"
+)
+
+//go:generate mockery --name ValidationWrapperFactory --structname ValidationWrapperFactoryMock --filename factory_mock.go --inpackage
+
+type ValidationWrapperFactory interface {
```

Comment:
Does it make sense to rename this as ValidationServiceFactory or AuthValidationFactory ?
I feel that `Wrapper` doesn't convey that much information.